scripts/yocto-build-and-test.sh

```diff
@@ -206,22 +206,14 @@ PREFERRED_VERSION${sep}pn-mender-configure = "$MENDER_CONFIGURE_MODULE_VERSION"
 EOF
     fi
 
-    if [ "${BUILD_CPP_CLIENT}" = "true" ]; then
-        BUILD_CPP_CLIENT_PATH=/src/github.com/mendersoftware/mender
-        ( cd ${WORKSPACE}/go/${BUILD_CPP_CLIENT_PATH} && git submodule update --init )
-        cat >> $BUILDDIR/conf/local.conf <<EOF
-PREFERRED_PROVIDER_virtual/mender-client = "mender"
-PREFERRED_PROVIDER_virtual/mender-client-native = "mender-native"
-EOF
-        # Remove after MEN-6279
-        cat >> $BUILDDIR/conf/local.conf <<EOF
+    # Remove after MEN-6279
+    cat >> $BUILDDIR/conf/local.conf <<EOF
 PREFERRED_VERSION:pn-mender-flash = "master-git%"
 EOF
-    fi
 
     cat >> $BUILDDIR/conf/local.conf <<EOF
-EXTERNALSRC${sep}pn-mender = "$WORKSPACE/go${BUILD_CPP_CLIENT_PATH:-}"
-EXTERNALSRC${sep}pn-mender-native = "$WORKSPACE/go${BUILD_CPP_CLIENT_PATH:-}"
+EXTERNALSRC${sep}pn-mender = "$WORKSPACE/go/src/github.com/mendersoftware/mender"
```

Comment:
Hmm, are you sure this will work? 

The CPP recipe does not create this hierarchy AFAIK

Reply:
It doesn't, but this is `EXTERNALSRC`, so it's not created by the recipe. It's created [by us](https://github.com/mendersoftware/mender-qa/blob/master/gitlab-pipeline/stage/init.yml#L124-L128).

We should ideally fix this in a better way, but that's for another day.

Reply:
Ahh, so that where it comes from :)